cypress/e2e/tests/migration/applicationinventory/assessment/miscellaneous.test.ts

```diff
@@ -231,6 +231,30 @@ describe(["@tier3"], "Tests related to application assessment and review", () =>
         archetypes[0].delete();
         deleteByList(tags);
     });
+    it("Deletes assessments from archived questionnaire associated with an archetype and an application", function () {
+        //automates polarion MTA-441 and MTA-442
+        const applications = createMultipleApplications(1);
+        const archetypes = createMultipleArchetypes(1);
+
+        AssessmentQuestionnaire.deleteAllQuestionnaires();
+        AssessmentQuestionnaire.enable(legacyPathfinder);
+        applications[0].perform_assessment("low", stakeholderList);
+        Application.open(true);
+        AssessmentQuestionnaire.disable(legacyPathfinder);
+        applications[0].validateAssessmentField("Unknown");
```

Comment:
2 additional checks need to be added , otherwise the PR LGTM.,

1)  After the questionnaire is disabled and before the assessment is deleted, the application 'assessment' status should show 'In progress' .

Reply:
done